docs/mixings.rst

```diff
@@ -0,0 +1,42 @@
+bayesmix/mixings
+
+Mixings
+=======
+
+In the algorithms of the library, we store a single ``Mixing`` object that represents the mixing measure used in the model.
```

Comment:
This is not completely accurate, see the documentation of `AbstractMixing` where we write

//! This class represents a prior for the mixture weights and the induced
//! exchangeable partition probability function (EPPF). See `ConditionalAlgorithm`
//! and `MarginalAlgorithm` for further details.